code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -46,6 +48,7 @@
 	desc = "This stops bleeding."
 	icon = 'icons/obj/surgery.dmi'
 	icon_state = "cautery"
+	var/overlay_state = "tray_cautery"
```

Comment:
Remove all of these variable definitions. They're unneeded now.

Reply:
Ok